### PR TITLE
[ADDED] cart model

### PR DESCRIPTION
**Pull request by Yshah112**

### CODE REVIEW

File: products/models.py [hunk 1]
```diff
@@ -102,3 +103,15 @@ class Meta:
 
     def __str__(self):
         return f'{self.banner_name}'
+
+
+class Cart(models.Model):
+    """this model is useful to add products into cart """
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
```
Comment (palakparmartntra) [inline products/models.py]: It's recommended to add a date and time field for auto-discarding functionality if we need to incorporate it.

Comment (Yshah112) [reply products/models.py]: done

File: products/models.py [hunk 2]
```diff
@@ -102,3 +103,16 @@ class Meta:
...
```
Comment (snehals21) [inline products/models.py]: @Yshah112 can we use models.PROTECT for user and product fields?

cc: @lalitvasoya

Comment (Yshah112) [reply products/models.py]: done

**Review by snehals21 — APPROVED** [review]

**Pull request merged**